Dockerfile

```diff
@@ -68,7 +64,7 @@ RUN apt-add-repository -y ppa:openjdk-r/ppa \
   && apt-get install -y --no-install-recommends openjdk-11-jdk \
   && apt-get install -y --no-install-recommends openjdk-17-jdk \
   && rm -rf /var/lib/apt/lists/* \
-  && update-alternatives --set java /usr/lib/jvm/java-11-openjdk-amd64/bin/java
+  && update-alternatives --set java /usr/lib/jvm/java-17-openjdk-amd64/bin/java
```

Comment:
Java 11 goes out of support soon, given we are doing an OS upgrade we may as well set the default Java to 17 at the same time